### PR TITLE
updated g meaning for 1.4.x

### PR DESCRIPTION
**Pull request by dgquintas**

### CODE REVIEW

**Comment by ctiller** [comment]
Also, regenerate projects.

**Comment by dgquintas** [comment]
Done.

File: doc/g_stands_for.md [hunk 1]
```diff
@@ -8,3 +8,4 @@ future), and the corresponding version numbers that used them:
 - 1.1 'g' stands for 'good'
 - 1.2 'g' stands for 'green'
 - 1.3 'g' stands for 'gentle'
+- 1.3 'g' stands for 'gregarious'
```
Comment (nicolasnoble) [inline doc/g_stands_for.md]: Is this numbering correct ?

**Comment by ctiller** [comment]
Even simple changes can be hard on a Friday afternoon :)

**Comment by dgquintas** [comment]
I can't even...

**Comment by dgquintas** [comment]
PTAL

**Comment by ctiller** [comment]
Now let's see which tests depend on the length of the user agent string.

**Comment by dgquintas** [comment]
... if tests decide to run

**Comment by dgquintas** [comment]
jenkins test this please

**Comment by dgquintas** [comment]
#10868 #9088

**Pull request merged**